docs/getting_started_guide/Installing_SIMP_using_r10k.rst

```diff
@@ -0,0 +1,263 @@
+  If your system does not have Internet access, you will need to adjust any paths
+  to point to your internal mirrors.
+
+.. NOTE::
+   This method does *not* modify your system's partitioning scheme or
+   encryption scheme to meet any regulatory policies. If you want an example of
+   what that should look like either see the :ref:`simp-installation-guide` or
+   check out the `Kickstart`_ files in the `simp-core Git repository`_.
+
+Installation of r10k
+^^^^^^^^^^^^^^^^^^^^
+
+On the system intended to be the Puppet server, run the following command to
+install the ``r10k`` ruby gem into the vendor ruby that comes with the
+```puppet-agent`` AIO package:
+
+.. code-block:: bash
+
+  $ /opt/puppetlabs/puppet/bin/gem install r10k
+
+``r10k`` can be used by calling the absolute path of the executable (unless
+added to ``$PATH``):
+
+.. code-block:: bash
+
+  $ /opt/puppetlabs/puppet/bin/r10k help
+
+
+Setting up your Control Repo
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SIMP uses a :term:`Puppetfile` and distributes some of a :term:`Control Repo` in
+the ``simp-environment`` package, or also in the `simp-core`_ repository.:
+
+On the filesystem of an installed SIMP system:
+
+.. code-block:: bash
+
+  $ tree -L 1 /usr/share/simp/environments/simp/
+  /usr/share/simp/environments/simp/
+  ├── environment.conf
+  ├── FakeCA
+  ├── hieradata
+  ├── manifests
+  └── modules
+
+Our git repo:
+
+.. code-block:: bash
+
+  $ tree -L 1 src/assets/simp-environment/environments/simp
+  src/assets/simp-environment/environments/simp
+  ├── environment.conf
+  ├── FakeCA/
+  ├── hieradata/
+  └── manifests/
+
+A Control Repo contains the modules, hieradata, and roles/profiles required for
+an infrastructure, and keeping it all in a git repo creates a workflow for
+updating and developing on your Puppet infrastructure.
+
+The modules are defined in a :term:`Puppetfile`, which SIMP uses as a development
+tool. We keep an up-to-date Puppetfile in the core of our repo, which you can
+download using this snippet:
+
+.. code-block:: bash
+
+  $ curl -o Puppetfile https://github.com/simp/simp-core/blob/<release>/Puppetfile.stable
+
+The example Puppetfile is labeled *stable*, meaning that the versions of the
+modules it contains are the ones contained in the last SIMP release. You can go
+to any previous release and download a Puppetfile with references to older
+modules from the git history of the ``simp-core`` repo.
+
+Our Puppetfile pulls down every dependency SIMP needs, which is currently
+contains more than just modules. Open up the ``Puppetfile`` that was just
+downloaded and remove the lines from ``moduledir 'src'`` to
+``moduledir 'src/puppet/modules'``. That should just leave the Puppet modules.
+
+Change into the directory you're using from above and run ``git init`` to create
+a control repo. Puppet, Inc has some great `documentation of a control repo online`_.
+
+Keydist, Rsync, and the Alternate Module Path
```

Comment:
I'm pretty sure that this is covered elsewhere and you might want to link to it.

Honestly, I'm not sure which is better. If you link, the information stays updated but they have to navigate around, which is never nice.